src/data/faq.json

```
@@ -117,6 +117,154 @@
+                        "anchor": "vac_cert_verification",
+                        "active": true,
+                        "textblock": [
+                            "The vaccination certificate (QR code) is used digitally via the CovPass-App or the Corona-Warn-App (CWA) or alternatively as a machine-readable print. The vaccination certificate contains only information about the vaccination status, the name of the vaccinated person, and the date of birth. For service providers who want to check the vaccination status, there will be a verification app to check the vaccination certificate. This allows the vaccination status to be scanned in a similar way to a barcode of a flight or train ticket. Alternatively, proof with the paper-based vaccination document is possible."
+                        ]
+                    },
+                    {
+                        "title": "Why has no joint EU-wide project been called out for proposal for a certificate?",
+                        "anchor": "vac_cert_eu",
+                        "active": true,
+                        "textblock": [
+                            "A joint EU call for proposals would have taken too much time and would have been difficult to implement due to the different vaccination information systems in the Member States. The EU approach is about regulating a recognition framework. Any cross-border pilots could be accompanied by the EU Commission. When implementing the Digital Proof of Vaccination in Germany, the EU requirements have been taken into account from the outset."
+                        ]
+                    },
+                    {
+                        "title": "Does my vaccination status affect the calculation of the risk? Does the calculation of the risk change if a vaccinated person has become infected, and then warns other users?",                        
+                        "anchor": "vac_cert_risk_assesment",
+                        "active": true,
+                        "textblock": [
+                            "Risk calculation and vaccination certificates are independent features of the app. The risk calculation does not take into account whether you are vaccinated against COVID-19. The calculation of the risk also does not change if a vaccinated person has become infected, and then warns other users."
+                        ]
+                    },
+                    {
+                        "title": "How to behave if I am partially or completely vaccinated, but receive a red warning in the Corona-Warn-App?",
+                        "anchor": "vac_cert_red_warning",
+                        "active": true,
+                        "textblock": [
+                            "Recommendation: If a vaccinated person has close contact with an infected person, no quarantine is necessary initially, as long as no symptoms appear. If symptoms do occur, you should get tested immediately and go into immediate quarantine."

```

I wouldn't use close contact but just high risk warning

```suggestion
                            "Recommendation: If a vaccinated person receives a high risk warning, no quarantine is necessary initially, as long as no symptoms appear. If symptoms do occur, you should get tested immediately and go into immediate quarantine."
```